tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -116,6 +116,13 @@ fn (mut app App) time() vweb.Result {
 In the future, Vweb will detect changes and recompile the website in the background
 while it's running.
 
+In the meantime, if you're using a *nix system, you can leverage `entr` to get a basic auto-reload environment going.
```

Comment:
V has its own sub command for that too: `v watch run blog.v`, that will restart the web server when you change something (it should work on windows too).

Reply:
Neat, thanks for the heads up @spytheman  :)

I understood the prior paragraph is not relevant anymore, so I removed it and updated my modest contribution :p